cea/default.config

```diff
@@ -379,6 +379,11 @@ type-scpanel.type = ChoiceParameter
 type-scpanel.choices = FP, ET
 type-scpanel.help = Type of solar collector: "FP" (Flat-plate) or "ET" (Evacuated-tubes).
 
+type-placement = OPT
+type-placement.type = ChoiceParameter
+type-placement.choices = OPT, USER
+type-placement.help = Panel placement method: either "OPT" for optimal angle and corresponding spacing, or "USER" for a user-defined angle and roof coverage.
```

Comment:
please specify that this only applies to panels on roof, but not panels on walls

Reply:
also, only possible on flat roofs?

Reply:
> also, only possible on flat roofs?
well, currently our code doesn't process anything _but_ flat roofs, right? so I can clarify that, but it's not like there is much of a difference :)

Reply:
rephrased as requested